llama_index/output_parsers/selection.py

```diff
@@ -45,7 +46,15 @@ class Answer(DataClassJsonMixin):
 
 
 class SelectionOutputParser(BaseOutputParser):
+    def _marshal_to_json(self, output: str) -> str:
+        "llm output to json friendly"
```

Comment:
could you make this docstring well-formatted? with three triple quotes on each side